libs/ui/src/components/molecules/Toast/Toast.tsx

```diff
@@ -38,7 +38,7 @@ type CustomToastProps = ToastProps &
     toastLinks?: ToastLinksProps;
   };
 
-type ToastLinksProps = {
+export type ToastLinksProps = {
```

Comment:
```suggestion
type ToastLinksProps = {
```
Don't think we're using ToastLinkProps anywhere outside this file.

Reply:
I'm using it in the context now. I can refactor it somewhere higher up.

Reply:
It looked like CustomToastProps was used in context. ToastLinkPros is a part of CustomToastProps.